doc/design/doi-generation-2.md

```diff
@@ -0,0 +1,87 @@
+# DOI for Draft Dandisets
+
+Author: Yaroslav O. Halchenko & Dorota Jarecka
+
+## The current approach
+
+- [initial design doc](./doi-generation-1.md)
+- overall:
+   - inject fake DOI upon dandiset creation
+   - mint proper DOI only upon dandiset publication (function `create_doi`)
+
+### Issues with the Existing Approach
+
+- [Stop injecting "fake" DOIs into draft dandisets](https://github.com/dandi/dandi-archive/issues/1709)
+- [Unpublished Dandisets display a DOI under `Cite As`](https://github.com/dandi/dandi-archive/issues/1932)
+
+## Proposed Solution
+
+Initially proposed/discussed in
+
+- [Create and maintain a "Findable" DOI for the Dandiset as a whole](https://github.com/dandi/dandi-archive/issues/1319)
+
+and boils down to the adoption of approach of Zenodo of having a DOI which always points to the latest version of the record.
```

Comment:
```suggestion
and boils down to the adoption of approach of Zenodo of having a "Concept DOI" which always points to the latest version of the record.
```

pre @asmacdo finding on the terminology of Zenodo. In our case it is more of a "Dandiset DOI" vs "Dandiset Published Version DOI"